.github/workflows/ci.yml

```diff
@@ -2,24 +2,24 @@ name: CI
 on:
   push:
     branches:
-    - src
+      - src
 
 jobs:
   CI:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
-    - uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: '2.7'
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "2.7"
 
-    - run: bundle install && bundle exec jekyll build
+      - run: bundle install && bundle exec jekyll build
 
-    - name: Deploy
-      uses: peaceiris/actions-gh-pages@364c31d33bb99327c77b3a5438a83a357a6729ad # v3.4.0
-      with:
-        github_token: ${{ secrets.GITHUB_TOKEN }}
-        publish_branch: master
-        publish_dir: _site
-        force_orphan: true
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@364c31d33bb99327c77b3a5438a83a357a6729ad # v3.4.0
```

Comment:
This also uses `node12` but it may not be as straightforward to fix.

Reply:
```suggestion
        uses: peaceiris/actions-gh-pages@4f9cc6602d3f66b9c108549d475ec49e8ef4d45e # v4.0.0
```